specifications/sep-payment-channel-transactions.md

```diff
@@ -565,6 +568,24 @@ process if its payment operation is dependent on amounts frozen.
 There is nothing participants can do to prevent this, other than using only auth
 immutable assets.
 
+### Trustline Limits
+
+Trustlines on the escrow accounts are defined as always having a maximum asset
```

Comment:
If my understanding is correct, escrow accounts will always have max limits, and closing transactions are made as claimable balances while retaining access of the escrow account to the initiator.

In none of these situations are we making a payment that can fail because of a lack of available limit, correct?

Reply:
Escrow accounts always have max limits, but there are no claimable balances in use. Payments occur directly between the two escrow accounts. Payments shouldn't fail unless they involve a trust line that has been auth revoked, or some of the asset in the sending trust line has clawed back.

Setting the asset limit to max should for the most part mean implementations don't need to check if the asset limit would be exceeded, assuming the accounts are not interacting with assets that have excessive supply on chain.

Reply:
ok that makes sense -- yeah I forgot we changed it to a two-escrow system